hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_unr_excl.el

```diff
@@ -174,521 +76,460 @@ ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (3 "11")
+Transition CheckFailError->FsmStateError "6->7"
+Fsm error_q "2210720134"
 ANNOTATION: "VC_COV_UNR"
-Condition 4 "2166812438" "((cmp_valid == CmpValid) ? err : ((cmp_valid == CmpInvalid) ? '0 : 1'b1)) 1 -1" (2 "1")
+Transition CheckFailError->AccessError "6->5"
+Fsm error_q "2210720134"
 ANNOTATION: "VC_COV_UNR"
-Condition 5 "2989213225" "((cmp_valid == CmpInvalid) ? '0 : 1'b1) 1 -1" (1 "0")
-CHECKSUM: "415438371 3779734926"
-INSTANCE: tb.dut.u_otp_ctrl_kdi.u_prim_count_entropy
+Transition AccessError->MacroEccCorrError "5->2"
+Fsm error_q "2210720134"
 ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (1 "01")
+Transition AccessError->CheckFailError "5->6"
+CHECKSUM: "158535765 1085514286"
+INSTANCE: tb.dut.u_otp_ctrl_dai
+Fsm error_q "1085514286"
 ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (3 "11")
+Transition FsmStateError->MacroEccCorrError "7->2"
+Fsm error_q "1085514286"
 ANNOTATION: "VC_COV_UNR"
-Condition 4 "2166812438" "((cmp_valid == CmpValid) ? err : ((cmp_valid == CmpInvalid) ? '0 : 1'b1)) 1 -1" (2 "1")
+Transition FsmStateError->AccessError "7->5"
+Fsm error_q "1085514286"
 ANNOTATION: "VC_COV_UNR"
-Condition 5 "2989213225" "((cmp_valid == CmpInvalid) ? '0 : 1'b1) 1 -1" (1 "0")
-CHECKSUM: "415438371 2692862965"
-INSTANCE: tb.dut.u_otp_ctrl_scrmbl.u_prim_count
+Transition AccessError->MacroEccCorrError "5->2"
+CHECKSUM: "3042386013 2710957581"
+INSTANCE: tb.dut.u_tlul_adapter_sram
 ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (1 "01")
+Condition 15 "1798941048" "(d_valid && d_error) 1 -1" (1 "01")
 ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (3 "11")
+Condition 9 "3130851184" "(vld_rd_rsp && reqfifo_rdata.error) 1 -1" (3 "11")
 ANNOTATION: "VC_COV_UNR"
-Condition 4 "2166812438" "((cmp_valid == CmpValid) ? err : ((cmp_valid == CmpInvalid) ? '0 : 1'b1)) 1 -1" (2 "1")
+Condition 8 "561780173" "((vld_rd_rsp && reqfifo_rdata.error) ? error_blanking_integ : (vld_rd_rsp ? rspfifo_rdata.data_intg : prim_secded_pkg::SecdedInv3932ZeroEcc)) 1 -1" (2 "1")
 ANNOTATION: "VC_COV_UNR"
-Condition 5 "2989213225" "((cmp_valid == CmpInvalid) ? '0 : 1'b1) 1 -1" (1 "0")
-CHECKSUM: "415438371 1726938517"
-INSTANCE: tb.dut.u_otp_ctrl_lfsr_timer.u_prim_count_integ
+Condition 6 "2745829604" "(vld_rd_rsp && reqfifo_rdata.error) 1 -1" (3 "11")
 ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (1 "01")
+Condition 5 "800561441" "((vld_rd_rsp && reqfifo_rdata.error) ? error_blanking_data : (vld_rd_rsp ? rspfifo_rdata.data : '0)) 1 -1" (2 "1")
+CHECKSUM: "3042386013 2558375765"
+INSTANCE: tb.dut.u_otp.gen_generic.u_impl_generic.u_tlul_adapter_sram
```

Comment:
No change, but the condition number changed, so if you are trying to find the items, you can just search the content.